Services/CO.CDP.OrganisationInformation.Persistence/Migrations/20250402144802_Add_Connected_Entities_Deleted_Column.cs

```diff
@@ -0,0 +1,698 @@
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace CO.CDP.OrganisationInformation.Persistence.Migrations
+{
+    /// <inheritdoc />
+    public partial class Add_Connected_Entities_Deleted_Column : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<List<string>>(
+                name: "deleted",
+                table: "connected_entities",
+                type: "boolean",
+                defaultValue: false,
+                nullable: false);
+
+            migrationBuilder.AddColumn<List<string>>(
+                name: "deleted",
+                table: "connected_entities_snapshot",
+                type: "boolean",
+                defaultValue: false,
+                nullable: false);
+
+            migrationBuilder.Sql($@"
+CREATE OR REPLACE PROCEDURE public.create_shared_consent_snapshot(
```

Comment:
At the point that this is merged - we will need to ensure any changes to this stored procedure are replayed over the top of the last migration that changed it.